aiida/orm/implementation/general/calculation/job/__init__.py

```diff
@@ -1773,7 +1773,7 @@ def _presubmit(self, folder, use_unstored_links=False):
         import os
         import json
 
-        from six.moves import cStringIO as StringIO
+        from six.moves import StringIO as StringIO
```

Comment:
no, we would like to use `cStringIO` on Python 2 (which is the default in Python 3 afaik)

Reply:
nevermind, sorry, just checked the other commit. Didn't know that cStringIO does not support unicode :+1:

Reply:
Python 2 has cStringIO (which is an implementation in C which only accepts ASCII strings) and StringIO which also accepts unicode. Python 3 only has io.StringIO which is optimised in C and is for text strings. I use plain StringIO here to try and simplify some of the encoding issues.
cStringIO should be faster for larger 'files' but introduces some problems with the move to unicode elsewhere in the code.